front/app/components/FormBuilder/edit/index.tsx

```diff
@@ -531,8 +531,7 @@ const FormBuilderPage = ({
     ? createPortal(
         <FormEdit
           defaultValues={{ customFields: formCustomFields }}
-          phaseId={phaseId}
-          projectId={projectId}
+          phase={phase.data}
```

Comment:
Can get phase + project id from the phase, and it's easier to work with phase in FormEdit if the type has already been narrowed down to `IPhaseData`.